platform/eureka/controller.go

```diff
@@ -0,0 +1,72 @@
+package eureka
+
+import (
+	"time"
+
+	"reflect"
+
```

Comment:
Please remove extra line between the imports. Lines between imports are used to group imports together.